.github/workflows/deploy_updated_changelog.yml

```
@@ -0,0 +1,178 @@
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Changelog version in major.minor format (e.g. '0.17'). Must match a file in config/changelogs/<version>[_flavor].md."
+        required: true
+        type: string
+      flavor:
+        description: "Optional app flavor for a flavor-specific changelog (leave blank to use the generic changelog for all flavors)."
+        required: false
+        type: choice
+        options:
+          - ""
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)